src/UIState/Wait.cpp

```diff
@@ -3,6 +3,13 @@
 #include "MainMenu.h"
 #include "TC_util.h"
 
+/**
```

Comment:
Nice documentation!

Reply:
Well I typed "/**" and VSCode did the rest...